repos/spack_repo/builtin/packages/exciting/package.py

```diff
@@ -115,7 +115,7 @@ def edit(self, spec, prefix):
             opts["BUILDMPI"] = "true"
             opts["MPIF90"] = spec["mpi"].mpifc
             opts["MPIF90_CPP_OPTS"] = "-DMPI -DMPIRHO -DMPISEC"
-            opts["MPIF90_OPTS"] = " ".join(["$(F90_OPTS)", "$(CPP_ON_OPTS) " "$(MPIF90_CPP_OPTS)"])
+            opts["MPIF90_OPTS"] = " ".join(["$(F90_OPTS)", "$(CPP_ON_OPTS) $(MPIF90_CPP_OPTS)"])
```

Comment:
I guess they meant a `,` here, but a static string suffices 😕 

```suggestion
            opts["MPIF90_OPTS"] = "$(F90_OPTS) $(CPP_ON_OPTS) $(MPIF90_CPP_OPTS)"
```